app/src/main/java/com/example/movie/FragmentMoviesList.kt

```diff
@@ -0,0 +1,44 @@
+package com.example.movie
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.cardview.widget.CardView
+import androidx.fragment.app.Fragment
+
+class FragmentMoviesList : Fragment() {
+
+    private var listener : MoviesListListener? = null
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // catch layout
+        return inflater.inflate(R.layout.fragment_movies_list, container, false)
+    }
+
+    override fun onAttach(context: Context) {
+        super.onAttach(context)
+        // catch listener
+        listener = context as? MoviesListListener
+    }
+
+    override fun onDetach() {
+        super.onDetach()
+        listener = null
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        view.findViewById<CardView>(R.id.card_movie).setOnClickListener {
+            listener?.toMovieDetail() }
```

Comment:
Старайся соблюдать форматирование кода. 
```suggestion
            listener?.toMovieDetail() 
        }
```